osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/MultiSpectatorScreen.cs

```diff
@@ -163,8 +164,11 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
+            Logger.Log($"{GetType().Name}.LoadComplete(): Resetting {nameof(MasterGameplayClockContainer)}.");
```

Comment:
Are all of log statements added in this file just for debug purposes? Were they meant to be committed here?

Reply:
Whoops.